src/endpoint/endpoint.js

```diff
@@ -8,215 +8,108 @@ require('../util/panic');
 console.log('loading .env file');
 require('../util/dotenv').load();
 
-const _ = require('lodash');
-const fs = require('fs');
-const url = require('url');
-const path = require('path');
-const util = require('util');
-const argv = require('minimist')(process.argv);
 const http = require('http');
 const https = require('https');
-const cluster = require('cluster');
-const size_utils = require('../util/size_utils');
 const FtpSrv = require('ftp-srv');
-const net = require('net');
-
+const url = require('url');
 const P = require('../util/promise');
-const promise_utils = require('../util/promise_utils');
 const dbg = require('../util/debug_module')(__filename);
-const api = require('../api');
-const config = require('../../config');
 const FuncSDK = require('../sdk/func_sdk');
 const ObjectIO = require('../sdk/object_io');
 const ObjectSDK = require('../sdk/object_sdk');
-const os_utils = require('../util/os_utils');
 const xml_utils = require('../util/xml_utils');
 const ssl_utils = require('../util/ssl_utils');
 const net_utils = require('../util/net_utils');
 const http_utils = require('../util/http_utils');
-
 const s3_rest = require('./s3/s3_rest');
 const blob_rest = require('./blob/blob_rest');
 const lambda_rest = require('./lambda/lambda_rest');
 const { FtpFileSystemNB } = require('./ftp/ftp_filesystem');
+const system_store = require('../server/system_services/system_store').get_instance();
+const md_server = require('../server/md_server');
```

Comment:
require md_server should change to register_common/object/func_services not in bare module

Reply:
Fixed with the last commit